### ISSUE TITLE
String parsing issue when using string macros and escaped quotes

### ISSUE THREAD

**Issue by alecloudenback** [julia-vscode/julia-vscode#1088]
Seems like when using string macros (e.g. RCall) and escaping the quote causes a parsing issue:

```julia
using RCall

R"paste(\"Hello\",\"World\",sep=\" \")"
```

![image](https://user-images.githubusercontent.com/711879/78061154-6196b280-7352-11ea-9b20-4387fe410abd.png)

With a little bit more complexity, I can get the parsing issue to extend beyond just the one line. Note the funny parsing of the triple quote:

![image](https://user-images.githubusercontent.com/711879/78061820-67d95e80-7353-11ea-99ec-54afb5f708e0.png)
```julia
using RCall

username = rcopy(R"keyring::key_list(\"database_name\")[1,2]")
password = rcopy(R"keyring::key_get(\"database_name\",username = keyring::key_list(\"database_name\")[1,2])");

x = """
    this string should be okay
    """
```

And how Juno parses for comparison:
![image](https://user-images.githubusercontent.com/711879/78061980-a0793800-7353-11ea-9905-d3fbbc1b4145.png)

**Comment by ZacLN** [comment]
I think this is a highlighting issue rather than parsing. We should be using the same highlighting spec as Juno so I'm not sure what's going on here...

**Comment by davidanthoff** [comment]
Yes, this needs to be fixed over in https://github.com/JuliaEditorSupport/atom-language-julia.
